app/views/merchant/dashboard/index.html.erb

```diff
@@ -1 +1,7 @@
 <h1> Welcome, <%= @merchant_user.name %>!</h1>
+<br>
+<center>
+  <h3><%= @merchant.name %></h3>
```

Comment:
At first, I thought Michael Scott should run a shop called "Mike's Bike Shop", but I realized that Michael could work for "Meg's Bike Shop" too. A little confusing, but I think this is okay :D

Reply:
Lmao this is called "Leiya is lazy and copied one of the shops from another page in the project -_-"